BSP/Inc/BSP_Timer.h

```diff
@@ -0,0 +1,5 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+int update_Time(int row);
+int writeReload(int reload);
```

Comment:
All functions should start with ```BSP_Timer_``` to make sure they are explicit. Also, make sure this API is common to both the simulator and the real board. 

We would want these functions probably:
```
BSP_Timer_Init()
BSP_Timer_Update()
```